packages/send-and-read-emails/frontend/react/src/SendEmails.jsx

```diff
@@ -83,45 +82,54 @@ function SendEmails({
   };
 
   return (
-    <form onSubmit={send} className="email-compose-view">
-      <h3 className="title">New Message</h3>
+    <form onSubmit={send} className={`email-compose-view ${variable}`}>
+      {!variable && <h3 className="title">New Message</h3>}
       <div className="input-container">
         <label className="input-label" htmlFor="To">
           To
         </label>
         <input
           aria-label="To"
+          type="email"
           value={to}
           onChange={(e) => setTo(e.target.value)}
         />
-        <div className="line"></div>
-
-        <label className="input-label" htmlFor="Subject">
-          Subject
-        </label>
-        <input
-          aria-label="Subject"
-          value={subject}
-          onChange={(e) => setSubject(e.target.value)}
-        />
-        <div className="line"></div>
+        {!variable && (
+          <>
+            <div className="line"></div>
+
+            <label className="input-label" htmlFor="Subject">
+              Subject
+            </label>
+            <input
+              aria-label="Subject"
+              value={subject}
+              onChange={(e) => setSubject(e.target.value)}
+            />
+            <div className="line"></div>
+          </>
+        )}
       </div>
       <textarea
         className="message-body"
         aria-label="Message body"
         placeholder="Type your message..."
-        rows={20}
+        rows={variable === 'small' ? 3 : 20}
```

Comment:
is `variable` just what dictates the body size?  We should probably give it a more descriptive name

Reply:
@gudsson Updated